src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example1.txt

```diff
@@ -1,28 +0,0 @@
-/*xml
-<module name="Checker">
-  <module name="TreeWalker">
-    <module name="SuppressionXpathSingleFilter">
-      <property name="files" value="File(One|Two)\.java"/>
-      <property name="checks" value="MethodName"/>
-      <property name="query" value="(//CLASS_DEF[@text='FileOne']/OBJBLOCK/
-                METHOD_DEF[@text='MyMethod']/IDENT)|
-                (//CLASS_DEF[@text='FileTwo']/OBJBLOCK/METHOD_DEF[@text='MyMethod']/IDENT)"/>
-    </module>
-  </module>
-</module>
-*/
```

Comment:
Why did we lose the config during transition to .java?
@Acesif , please go through the description again.. You need to understand how our parser works.